packages/web/src/scss/components/TextField/index.html

```diff
@@ -98,6 +98,36 @@
       </button>
     </div>
   </div>
+  <div class="TextField TextField--danger">
+    <label
+      for="textfieldPasswordDangerMessage"
+      class="TextField__label TextField__label--required"
+    >Password Toggle</label>
+    <div class="TextField__passwordToggle">
+      <input type="password" id="textfieldPasswordDangerMessage" class="TextField__input" value="Danger" />
+      <button
+        type="button"
+        class="TextField__passwordToggle__button"
+        role="switch"
+        aria-pressed="false"
+        aria-label="Show password"
+        data-toggle="password"
+      >
+        <span class="TextField__passwordToggle__icon TextField__passwordToggle__icon--hidden">
+          <svg class="Icon" width="24" height="24" aria-hidden="true">
```

Comment:
Can you please remove the `Icon` class? It does nothing, I just forgot to remove it earlier.

Reply:
done